src/assets/styles/global-styles.js

```diff
@@ -59,8 +59,14 @@ ${fonts}
   a {
     text-decoration: none;
     transition: color 0.05s ease-in-out;
+
```

Comment:
what about not visited link colour? :)

Reply:
I've removed `color: inherit` from visited link and moved it to an `a` element. Now it works correctly :)